network_importer/config.py

```diff
@@ -64,6 +64,9 @@ class NetboxSettings(BaseSettings):
     token: Optional[str]
     verify_ssl: bool = True
 
+    model_flag_tags: Optional[list] # List of tags that defines what objects to assign the model_flag_mode to.
+    model_flag_mode: Optional[str]  # The model flag that will be applied to objects based on tag.
```

Comment:
The DiffSync flags are ints (enums), not strings.

Reply:
Per our Zoom discusison - `Optional[DiffSyncModelFlags]` is probably actually more correct here, but will require some investigation to determine how/whether we can set enum values in a `.toml` file and whether Pydantic will accept integer values as inputs if not.

Reply:
On testing int() works well.